VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddResponseFragment.java

```diff
@@ -62,7 +62,7 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
         textViewTitle.setText(discussionTopic.getTitle());
         textViewResponse.setText(Html.fromHtml(discussionTopic.getRenderedBody()));
-        textViewTimeAuthor.setText(DiscussionTextUtils.getAuthorAttributionText(discussionTopic, getResources()));
+        textViewTimeAuthor.setText(DiscussionTextUtils.getAuthorAttributionText(discussionTopic, getResources(), discussionTopic.getType() == DiscussionThread.ThreadType.DISCUSSION ? DiscussionTextUtils.AuthorAttributionType.DISCUSSION : DiscussionTextUtils.AuthorAttributionType.QUESTION));
```

Comment:
Could we have a utility method in `DiscussionThread` that returns the `AuthorAttributionType` based on this logic? Also, this line is a quite long, maybe it should be split into two lines.